src/app/(content)/courses/[course]/[lesson]/page.tsx

```diff
@@ -0,0 +1,50 @@
+import {loadLesson} from '@/lib/lessons'
+import {loadCourse} from '@/lib/courses'
+import {LessonResource} from '@/types'
+import {notFound} from 'next/navigation'
+import {PlayerTwo} from '@/app/(content)/courses/[course]/[lesson]/Player'
```

Comment:
hadn't seen the `(content)` route grouping before but I like it

https://nextjs.org/docs/app/building-your-application/routing/route-groups#examples